bsmetadata/preprocessing_tools/entity_desc_utils.py

```diff
@@ -0,0 +1,29 @@
+import re
+import string
+from collections import defaultdict
+
+import nltk
+from wikipedia2vec.dump_db import DumpDB
+
+
+class EntityDescUtils:
```

Comment:
Since the same code is also being used by Website Utils, I think we can merge them together to have a common utility class and use that instead

Reply:
If there are no other issues apart from this, can we address this in a separate PR due to other priority tasks on my to-do?

Reply:
Fixed!